client-api/src/main/java/org/sonarsource/sonarlint/core/clientapi/backend/issue/ResolutionStatus.java

```diff
@@ -19,15 +19,15 @@
  */
 package org.sonarsource.sonarlint.core.clientapi.backend.issue;
 
-public enum IssueStatus {
+public enum ResolutionStatus {
```

Comment:
Be aware that this will break the clients' update of the CORE. I think it's ok, not a big change, but it might be good to communicate the changes to all parties